features/statistic/metric_percentage.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Calculating the percentage of metrics used
+  In order to be able to calculate the metric percentage used
+  As a developer
+  I want to know which metrics are more used
+
+
+  Scenario: With an used metric
```

Comment:
Don't forget about the `@kalibro_configuration_restart` otherwise a tests following this will fail since the KalibroConfiguration name will be in use.